_git/git_commands.py

```diff
@@ -0,0 +1,434 @@
+from aenea import Text
+
+'''
+NOTE: Not all possible commands and options are available in this grammar.
+NatLink puts a limit on how complex the grammar can get, and some commands had
+to be removed. If you feel like there is something missing here, feel free to
+add stuff back and make a pull request.
+
+NOTE 2: Lists of options were grepped from the Git help pages using a bash
+script. These have been marked with the comment 'Generated'. Some of these
+generated lists are too long, so had to be shortened (see note above). You can
+find the script here in case you find it useful
+https://gist.github.com/dylan-chong/ecf701b1a623c9f2ccb78cbb5db700c6 . Ideally,
+when adding a new command, just add the options that are commonly used.
+'''
```

Comment:
Regarding this note, there are various generated lists of options here, but I haven't had time to trim them down. There are by no means too many options for Nat link to handle, but it looks rather silly adding all of these commands that probably no one will ever use. It will take a very long time to remove all of the not useful ones, which I will do over time and submit another pull request sometime in the future

Reply:
I think this is fine. I don't have great insight into how people use Aenea (perhaps I should try a survey at some point), but I would guess that 100% voice controlled is less important than handling the majority of common cases to greatly cut down on the need to type.

So in general I would favor prioritizing common cases, especially with something like git which has a tremendous amount of complex options.

I'm open to other opinions here.

Reply:
> I would guess that 100% voice controlled is less important than handling the majority of common cases to greatly cut down on the need to type.

I definitely agree with you there